api/tests/unit/domain/read-models/ComplementaryCertificationCourseResultsForJuryCertification_test.js

```diff
@@ -0,0 +1,271 @@
+      },
+      {
+        partnerKey: 'PIX_EDU_FORMATION_INITIALE_1ER_DEGRE_CONFIRME',
+        acquired: false,
+        source: 'PIX',
+        expectedSource: 'PIX',
+        expectedStatus: 'Rejetée',
+        expectedLabel: 'Pix+ Édu Confirmé',
+      },
+      {
+        partnerKey: 'PIX_EDU_FORMATION_CONTINUE_1ER_DEGRE_CONFIRME',
+        acquired: true,
+        source: 'PIX',
+        expectedSource: 'PIX',
+        expectedStatus: 'Validée',
+        expectedLabel: 'Pix+ Édu Confirmé',
+      },
+      {
+        partnerKey: 'PIX_EDU_FORMATION_CONTINUE_1ER_DEGRE_CONFIRME',
+        acquired: false,
+        source: 'PIX',
+        expectedSource: 'PIX',
+        expectedStatus: 'Rejetée',
+        expectedLabel: 'Pix+ Édu Confirmé',
+      },
+      {
+        partnerKey: 'PIX_EDU_FORMATION_CONTINUE_1ER_DEGRE_AVANCE',
+        acquired: true,
+        source: 'EXTERNAL',
+        expectedSource: 'EXTERNAL',
+        expectedStatus: 'Validée',
+        expectedLabel: 'Pix+ Édu Avancé',
+      },
+      {
+        partnerKey: 'PIX_EDU_FORMATION_CONTINUE_1ER_DEGRE_AVANCE',
+        acquired: false,
+        source: 'PIX',
+        expectedSource: 'PIX',
+        expectedStatus: 'Rejetée',
+        expectedLabel: 'Pix+ Édu Avancé',
+      },
+      {
+        partnerKey: 'PIX_EDU_FORMATION_CONTINUE_1ER_DEGRE_EXPERT',
+        acquired: true,
+        source: 'PIX',
+        expectedSource: 'PIX',
+        expectedStatus: 'Validée',
+        expectedLabel: 'Pix+ Édu Expert',
+      },
+      {
+        partnerKey: 'PIX_EDU_FORMATION_CONTINUE_1ER_DEGRE_EXPERT',
+        acquired: false,
+        source: 'PIX',
+        expectedSource: 'PIX',
+        expectedStatus: 'Rejetée',
+        expectedLabel: 'Pix+ Édu Expert',
+      },
+    ].forEach(({ partnerKey, acquired, source, expectedStatus, expectedLabel, expectedSource }) => {
+      describe(`when partnerKey is ${partnerKey} and acquired is ${acquired}`, function () {
+        it(`should return a ComplementaryCertificationCourseResultsForJuryCertification with status "${expectedStatus}" and label "${expectedLabel}"`, function () {
+          // when
+          const complementaryCertificationCourseResultsForJuryCertification =
+            ComplementaryCertificationCourseResultsForJuryCertification.fromComplementaryCertificationCourseResult({
+              partnerKey,
+              acquired,
+              source,
+              complementaryCertificationCourseId: 123,
+            });
+
+          // then
+          expect(complementaryCertificationCourseResultsForJuryCertification.status).equal(expectedStatus);
+          expect(complementaryCertificationCourseResultsForJuryCertification.label).equal(expectedLabel);
+          expect(complementaryCertificationCourseResultsForJuryCertification.source).equal(expectedSource);
+          expect(complementaryCertificationCourseResultsForJuryCertification.complementaryCertificationCourseId).equal(
+            123
+          );
```

Comment:
suggestion: plus concis:
```suggestion
          expect(complementaryCertificationCourseResultsForJuryCertification).to.deep.equal({
            status: expectedStatus,
            label: expectedLabel,
            source: expectedSource,
            complementaryCertificationCourseId: 123,
          });
```